test/docfx.Test/TestUtility.cs

```diff
@@ -66,7 +66,7 @@ void Throw(string stackTrace, string message, string detailMessage, string info)
 
         private static void CreateZipFile(KeyValuePair<string, string> file, string filePath)
         {
-            var token = YamlUtility.ToJToken(file.Value);
+            var token = YamlUtility.ToJToken(new StringReader(file.Value));
```

Comment:
why not put `new StringReader()` into `ToJToken` as well, or another overloading for it?

Reply:
Either is ok, `ToJToken` is more of a private method, I initially extract it to share between docfx and docastest. Didn't thought it is used here. This line can call `Deserialize<JToken>` instead